### ISSUE TITLE
[0.5.0] Limit to Date object only

### ISSUE THREAD

**Issue by SamBrishes** [pytesNET/tail.DateTime#50]
Hellow,

just a small question on this table: I'm planing to use `new Date( )` objects for ANY developer/user-passed Date/Time inputs only. The current version allows passing strings or timestamps, which may gets interpret correctly, but in the most time it will fail. The reason is dead simple... There are too many ways to format a Date string: ``MM/DD/YYYY``, `DD/MM/YYYY`, `DD.MM.YYYY`, `YYYY-MM-DD`, `12. January 2020`, `January 12, 2020` just to name a few ones.

The `tail.DateTime` script shouldn't have the task to interpret and parse a Date/Time string (just to follow the UNIX way to code). That's one reason [`moment.js`](https://momentjs.com/) and his configurable and extensive Date/Time string parser is such popular. 

Of course, there is also this "small" alternative [`date-fns`](https://date-fns.org/) package, but this supports ISO 8601 formatted Date/Time strings only, and [`datejs`](https://github.com/datejs/Datejs) although has a good inventory but extends / replaces the Core object (and isn't maintained anymore, I guess).

It is a really tricky stuff to parse user inputs correctly, since we live all around this planet using all our own written Date/Time outputs.

# The Solution
In other words, I want to "move" this problem to the end user (You ;D) by...

- ... allowing `new Date( ... )` objects on the configurations ONLY
- ... offering a callback method to parse user-values (passed through the `<input />` field)
- ... **optional** supporting `moments.js` to parse user-values (passed through the `<input />` field)

"Optional", because my `tail.` project series should be as dependency-less as possible and I don't want to require the user to download x packages to use mine. But supporting optional ones (such as on my `tail.writer` script) would just allow the user to extend the functionality on his own.

What do you think?

Sincerely,
Sam.
